src/other/global.sh

```diff
@@ -1,8 +1,13 @@
 #!/bin/bash
+
+GREEN="\e[1;32;49m"
+NORMAL="\e[0m"
+RED="\e[1;91;49m"
```

Comment:
Ok I think that this solution definitely solves the problem with termux is solved. :ok_hand: